src/main/resources/application.properties

```
@@ -2,13 +2,20 @@ application.namespace="presenter-account-consumer"
 api.base.path=${API_URL}
 internal.api.base.path=${INTERNAL_API_URL}
 chs.internal.api.key=${CHS_INTERNAL_API_KEY}
+
 spring.datasource.url=${XML_DATASOURCE_URL}
 spring.datasource.username=${XML_DATASOURCE_USERNAME}
 spring.datasource.password=${XML_DATASOURCE_PASSWORD}
 spring.datasource.driver.class=oracle.jdbc.driver.OracleDriver
 spring.datasource.hikari.maximum-pool-size=${XML_DATASOURCE_MAX_POOL_SIZE}
+
+spring.jpa.hibernate.ddl-auto=none
+spring.jps.database-platform=org.hibernate.dialect.OracleDialect
```

Should this be `jpa`?
```suggestion
spring.jpa.database-platform=org.hibernate.dialect.OracleDialect
```